test/controllers/api/resource_controller_test.rb

```diff
@@ -285,6 +285,25 @@ class Api::ResourceControllerTest < ActionDispatch::IntegrationTest
     assert_equal response.parsed_body["data"].pluck("id").map(&:to_i).sort, [@api_resource_1.id, @api_resource_3.id].sort
   end
 
+  test '#index search jsonb field - string - partial: multi word string' do
+    @api_resource_1.update(properties: {name: 'Professional Writer'})
+    @api_resource_2.update(properties: {name: 'Physical Development'})
+    @api_resource_3.update(properties: {name: 'Professional Development'})
+
+    payload = { 
+      properties: { 
+        name: { 
+          value: 'professional development',
+          option: 'PARTIAL'
```

Comment:
our use case for `PARTIAL` is something like this `%professional development%`. so adding a new matcher for `KEYWORDS` would be better where, 

1. if a string is passed ("Pro development") => "pro" and "development"
2. if an array of strings are passed `['pro', 'dev']` => "pro" and "dev"

Reply:
> if an array of strings are passed ['pro', 'dev'] => "pro" and "dev"
@donrestarone For confirmation;

For ['pro dev', 'test'], both 'pro' & 'dev' should match, right?

Reply:
yup! @alis-khadka